README.md

```diff
@@ -1,63 +1,79 @@
+
+## Running the Superpeer
+
+Superpeer is a Gradle application - it can be built with `gradlew build`, and a binary can be generated with `gradlew installDist`.
+
+If run without arguments Superpeer polls STDIN for input - typing `exit` will shut down RightMesh and stop the application. Unless you are developing/debugging, you will likely want to run Superpeer with the `-h | --headless` flag, which doesn't poll for input and responds to SIGINT signals (e.g. can be killed cleanly with `Ctrl+C` or task managers).
+
+## Connecting to your Superpeer
+
+By default RightMesh devices with internet connections will connect to a Superpeer operated by RightMesh at `research.rightmesh.io`. In the future the goal is to have a network of Superpeers, with Superpeers implemented and operated by both RightMesh and community members.
+
+If you would like to have devices use your Superpeer instance for testing, you can hard-code its address into a RightMesh application to make it use your instance instead of the production network. You can do this by passing the address as a parameter to a special MeshManager constructor that is available in the developer version of the library. Note that this will never be an option in production - all RightMesh applications will use the same network, so that the owner of the device can specify their preferences for where/when/how to connect to Superpeers.
 
-Todo: detect in gradle or use a set variable to specify whether we are
-running on a local machine or AWS instance. Also add an option to set
-a remote parity machine (if the superpeer wants to separate funciton).
+If you would like to send data to a Java application that makes use of RightMesh in a production application, you will need to find out its MeshID and send data to it through the RightMesh library.
 
-# Steps to run on AWS
-Install and get parity running
-`./configure.sh`
+## Running the Superpeer as a Service
 
-Verify parity is running:
-`ps ax | grep parity`
+This repo contains everything needed to set up a Superpeer as a service on Ubuntu. These steps will help you get started:
 
-Compile against the latest library version
-`./gradlew build --refresh-dependencies`
+1. Set up an Amazon EC2 Instance running Ubuntu
+    - A Micro instance is fine, or any other Ubuntu server
+    - Make sure port 22/TCP is open for SSH, 40,000/UDP for the Superpeer, and both 30303/TCP and 30303/UDP for Parity to communicate with the Ethereum network.
+2. SSH Into the server.
+    - `ssh ubuntu@your.ip.address.here`
+3. Install Java
+    - `sudo add-apt-repository ppa:webupd8team/java`
+    - `sudo apt-get update`
+    - `sudo apt-get install oracle-java8-installer`
+4. Clone the Superpeer repository and navigate to the directory.
+    - `git clone https://github.com/rightmesh/superpeer`
+    - `cd superpeer`
+5. Update build.gradle with your credentials from the [RightMesh developer portal](https://developer.rightmesh.io/keys/).
+6. Install and configure Parity
```

Comment:
Should we mention explicitly the parity version here? Also sidenote - who makes sure we keep up with parity updates in this repo?